packages/universal-router/src/utilities.ts

```diff
@@ -0,0 +1,3 @@
+export function isServer() {
```

Comment:
I assume this is just for mocking in tests?

Reply:
Yeah, thats right.